Documentation/api-reference.md

```diff
@@ -166,6 +161,49 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
   end sub, context)
 ```
 
+#### Send Custom IdentityMap:
+
+`sendEvent` API allows passing custom identifiers to the Edge Network using custom Identity map . Create the map using the identifier namespace as key and pass in the identity items for the namespace as an array. Configure the "primary" and "authenticatedState" per individual identity item per your application's requirements.
+
+> **Note**
+> Passing custom Identity map is optional. Do not pass the ECID with the sendEvent API, the ECID automatically attaches it on all requests. By default, the ECID is set as primary server-side if no other identifier is set uses "primary" : true.
+
+```brightscript
+customIdentityMap = {
+    "RIDA" : [
+          {
+            "id" : "SampleAdIdentifier",
+            "authenticatedState": "ambiguous",
+            "primary": false
+          }
+    ],
+    "EMAIL" : [
+          {
+            "id" : "user@example.com",
+            "authenticatedState": "ambiguous",
+            "primary": false
+          },
+          {
+            "id" : "useralias@example.com",
+            "authenticatedState": "ambiguous",
+            "primary": false
+          }
+    ]
+}
+```
+
+##### Sample:
```

Comment:
- can we move this title before the id creation and club the two into one
- can we use `Example` instead of `Sample` similar with other platform docs
reading:

**Example**
```
// create the custom IdentityMap
...

// send Experience event with IdentityMap
...
```